### PR TITLE
[#162414225] Setup travis CI integration 

### PR DESCRIPTION
**Pull request by Sammuel09**
#### What does this PR do?
* Integrate Continuous Integration with Travis CI

#### Description of Task to be completed?
* Connect the Github repository to Travis CI
* Configure the `travis.yml` file. 

#### How should this be manually tested?
* N/A

#### What are the relevant pivotal tracker stories?
[ Integrate Continuous Integration with Travis CI](https://www.pivotaltracker.com/n/projects/2229851/stories/162414225)

### CODE REVIEW

**Comment by rafmme** [comment]
LGTM, yet

**Comment by Bobsar0** [comment]
LGTM, yet

**Comment by tejiri4** [comment]
LGTM, yet.

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,11 @@
+node_modules/
+#Ignore dist folder
```
Comment (Bobsar0) [inline .gitignore]: Remove this comment

Comment (Sammuel09) [reply .gitignore]: Ok. Thanks Steve.

**Pull request merged**